paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/playground/settings/NewUpdateCardScreenDefinition.kt

```diff
@@ -0,0 +1,15 @@
+package com.stripe.android.paymentsheet.example.playground.settings
+
+import com.stripe.android.core.utils.FeatureFlags
+import com.stripe.android.paymentsheet.example.playground.model.CheckoutRequest
+
+internal object NewUpdateCardScreenDefinition : BooleanSettingsDefinition(
+    key = "new_update_card_screen",
+    displayName = "Enable new update card screen",
+    defaultValue = false,
+) {
+    override fun configure(value: Boolean, checkoutRequestBuilder: CheckoutRequest.Builder) {
```

Comment:
You probably want `valueUpdated` instead. This is meant to customize the checkout request.